sdk/anomalydetector/Azure.AI.AnomalyDetector/samples/Sample4_MultivariateDetect.md

```diff
@@ -103,7 +103,10 @@ private MultivariateDetectionResult BatchDetect(AnomalyDetectorClient client, Ur
     try
     {
         Console.WriteLine("Start batch detect...");
-        MultivariateBatchDetectionOptions request = new MultivariateBatchDetectionOptions(datasource, 10, startTime, endTime);
+        MultivariateBatchDetectionOptions request = new MultivariateBatchDetectionOptions(datasource, startTime, endTime)
```

Comment:
the ctor we previously were using is internal.
We should never do that because our users are not able to use internal ctors.
Here we change the sample to use the public ctor